### PR TITLE
New Resource: azurerm_bot_channel_facebook

### PR DESCRIPTION
**Pull request by neil-yechenwei**
Currently, TF doesn't support Bot Facebook Channel. So I submitted this PR to implement it.

Please refer this [doc](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-channel-connect-facebook?view=azure-bot-service-4.0) to create test data for facebook if need be.

--- PASS: TestAccBotChannelsRegistration/channel/facebookBasic (214.49s)
--- PASS: TestAccBotChannelsRegistration/channel/facebookUpdate (335.30s)
--- PASS: TestAccBotChannelsRegistration/channel/facebookRequiresImport (261.06s)

API reference:
https://github.com/Azure/azure-rest-api-specs/blob/b20011746ef065a8ab6572045de9578c192f2ffd/specification/botservice/resource-manager/Microsoft.BotService/stable/2021-03-01/botservice.json#L1437

### CODE REVIEW

File: website/docs/r/bot_channel_facebook.markdown [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+The following arguments are supported:
+
+* `resource_group_name` - (Required) The name of the resource group where the Facebook Channel should be created. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `bot_name` - (Required) The name of the Bot Resource this channel will be associated with. Changing this forces a new resource to be created.
+
+* `app_id` - (Required) The Facebook Application ID for the Facebook Channel.
```
Comment (katbyte) [inline website/docs/r/bot_channel_facebook.markdown:57]: lets be explicit with what this is?
```suggestion
* `facebook_application_id` - (Required) The Facebook Application ID for the Facebook Channel.
```

File: website/docs/r/bot_channel_facebook.markdown [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+* `app_secret` - (Required) The Facebook Application Secret for the Facebook Channel.
```
Comment (katbyte) [inline website/docs/r/bot_channel_facebook.markdown:59]: and here
```suggestion
* `facebook_application_secret` - (Required) The Facebook Application Secret for the Facebook Channel.
```

**Review by katbyte — CHANGES_REQUESTED** [review]

File: website/docs/r/bot_channel_facebook.markdown (hunk 1 above)
Comment (neil-yechenwei) [reply website/docs/r/bot_channel_facebook.markdown:57]: Updated

File: website/docs/r/bot_channel_facebook.markdown (hunk 2 above)
Comment (neil-yechenwei) [reply website/docs/r/bot_channel_facebook.markdown:59]: Updated

**Comment by neil-yechenwei** [comment]
@katbyte , thanks for your comments. I've updated code. Please have an another look. Thanks.

**Review by katbyte — APPROVED** [review]
Thanks @neil-yechenwei - LGTM 🚀

**Pull request merged**